### ISSUE TITLE
Add the ability to see the list of all sites you've enabled this on

### ISSUE THREAD

**Issue by Victor239** [denschub/chrome-mask#11]
Rather than having to navigate to sites you've been to previously to check whether you enabled it on there or not, it would be good to see the centralised list and be able to remove or add entries as needed.

**Comment by denschub** [comment]
Yeah, fair request. I'll add something eventually.

As a workaround for now, you can always open `about:debugging`, select "This Firefox", click the "Inspect" button next to Chrome Mask, then select "Storage", then "Extension Storage". You'll find a list of sites in the `enabledHostnames` entry. Careful when editing this, though, as the addon fails in unexpected way if that's not a valid array. :)

**Comment by denschub** [comment]
Sorry for the late response. I don't have any preferences - just a simple list in the addon's preference section on `about:addons` should be fine. Maybe that's hard to discover for some, but we can improve that problem if we know we actually run into it. :)

**Comment by dannycolin** [comment]
I was going to include a button in the popup that redirect the user to the option page.

**Comment by denschub** [comment]
I'm a bit worried about visual noise - but I trust you, and will have a look at whatever you come up with :)

**Comment by ConnoisseurProtege** [comment]
To add to this, would it be possible to  add a toggle to enable chrome mask automatically on every visited website and whitelist the ones you don't want it enabled on?

**Comment by denschub** [comment]
No. "Spoofing by default on all websites" does more harm than good, and it's not a use case I will ever support, sorry.

**Comment by ConnoisseurProtege** [comment]
> Spoofing by default on all websites" does more harm than good

I see, I'm not a very technical person, I would like to learn the issues it causes if you don't mind? I don't want to use the extension irresponsibly as I've just been enabling it on every website

**Comment by denschub** [comment]
Please see https://github.com/denschub/chrome-mask/discussions/26 - and if you have questions, let's continue there so we can keep this issue clean :)

**Comment by denschub** [comment]
#33 is now merged, which adds a UI that shows a list of all enabled sites, and allows you to add/remove sites. I'll release this soon, but I'll close this issue anyway.
